common/dtd/ldml.xsd

```diff
@@ -0,0 +1,8624 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright © 1991-2023 Unicode, Inc.
+  For terms of use, see http://www.unicode.org/copyright.html
+  SPDX-License-Identifier: Unicode-DFS-2016
+  CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+-->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
+  <xs:import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="xml.xsd"/>
+  <xs:element name="ldml">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element ref="identity"/>
+        <xs:choice>
+          <xs:element ref="alias"/>
+          <xs:sequence>
+            <xs:element minOccurs="0" maxOccurs="unbounded" ref="fallback"/>
+            <xs:element minOccurs="0" ref="localeDisplayNames"/>
+            <xs:element minOccurs="0" ref="layout"/>
+            <xs:element minOccurs="0" ref="contextTransforms"/>
+            <xs:element minOccurs="0" ref="characters"/>
+            <xs:element minOccurs="0" ref="delimiters"/>
+            <xs:element minOccurs="0" ref="measurement"/>
+            <xs:element minOccurs="0" ref="dates"/>
+            <xs:element minOccurs="0" ref="numbers"/>
+            <xs:element minOccurs="0" ref="units"/>
+            <xs:element minOccurs="0" ref="listPatterns"/>
+            <xs:element minOccurs="0" ref="collations"/>
+            <xs:element minOccurs="0" ref="posix"/>
+            <xs:element minOccurs="0" ref="characterLabels"/>
+            <xs:element minOccurs="0" ref="segmentations"/>
+            <xs:element minOccurs="0" ref="rbnf"/>
+            <xs:element minOccurs="0" ref="typographicNames"/>
+            <xs:element minOccurs="0" ref="personNames"/>
+            <xs:element minOccurs="0" ref="annotations"/>
+            <xs:element minOccurs="0" ref="metadata"/>
+            <xs:element minOccurs="0" ref="references"/>
+            <xs:element minOccurs="0" maxOccurs="unbounded" ref="special"/>
+          </xs:sequence>
+        </xs:choice>
+      </xs:sequence>
+      <xs:attribute name="version"/>
+      <xs:attribute name="draft">
+        <xs:simpleType>
+          <xs:restriction base="xs:token">
+            <xs:enumeration value="approved"/>
+            <xs:enumeration value="contributed"/>
+            <xs:enumeration value="provisional"/>
+            <xs:enumeration value="unconfirmed"/>
+            <xs:enumeration value="true"/>
+            <xs:enumeration value="false"/>
+          </xs:restriction>
+        </xs:simpleType>
+      </xs:attribute>
+    </xs:complexType>
+  </xs:element>
+  <!-- @METADATA -->
+  <!-- @DEPRECATED -->
+  <!-- @METADATA -->
```

Comment:
Out of curiosity, what does it mean to have the two successive `@METADATA/@DEPRECATED` annotations here?

Reply:
These are now removed.